### PR TITLE
Fixed indentation and added Android to list of platforms os.

### PR DESCRIPTION
**Pull request by Lunarsong**

### CODE REVIEW

**Comment by Lunarsong** [comment]
I signed it! :)

File: site/docs/platforms.md [hunk 1]
```diff
@@ -102,9 +102,10 @@ and operating systems.
    *  `@bazel_tools//platforms:ppc`
    *  `@bazel_tools//platforms:arm`
    *  `@bazel_tools//platforms:s390x`
-*   `@bazel_tools//platforms:os` defines the following operating systems:
+*  `@bazel_tools//platforms:os` defines the following operating systems:
    *  `@bazel_tools//platforms:osx`
```
Comment (laszlocsomor) [inline site/docs/platforms.md]: Please also add `@bazel_tools//platforms:ios` and sort the list.

File: site/docs/platforms.md [hunk 2]
```diff
@@ -102,9 +102,10 @@ and operating systems.
...
    *  `@bazel_tools//platforms:freebsd`
+   *  `@bazel_tools//platforms:android`
    *  `@bazel_tools//platforms:linux`
    *  `@bazel_tools//platforms:windows`
```
Comment (laszlocsomor) [inline site/docs/platforms.md]: Could you please add a comment below saying:

> You can see this list by running:
> ```
> bazel query 'attr(constraint_setting, "@bazel_tools//platforms:os", "@bazel_tools//platforms:*")'
> ```

File: site/docs/platforms.md [hunk 3]
```diff
@@ -102,9 +102,10 @@ and operating systems.
    *  `@bazel_tools//platforms:ppc`
    *  `@bazel_tools//platforms:arm`
    *  `@bazel_tools//platforms:s390x`
```
Comment (laszlocsomor) [inline site/docs/platforms.md]: Could you please add a comment below saying:

> You can see this list by running:
> ```
> bazel query 'attr(constraint_setting, "@bazel_tools//platforms:cpu", "@bazel_tools//platforms:*")'
> ```

**Comment by irengrig** [comment]
Hi @Lunarsong, thank you for your contribution!
Bazel sheriff here.
What is the status of this pull request? Should we take any actions here?

**Pull request closed without merging**